packages/types/src/legacy/reference-config.ts

```diff
@@ -69,305 +67,7 @@ export type ThemeId =
-   */
-  generateWebhookSlug?: ((webhook: { name: string; method?: string }) => string) | undefined
-  /**
-   * The baseServerURL is used when the spec servers are relative paths and we are using SSR.
-   * On the client we can grab the window.location.origin but on the server we need
-   * to use this prop.
-   *
-   * @default undefined
-   * @example 'http://localhost:3000'
-   */
-  /**
-   * Callback that triggers as soon as the references are lazy loaded.
-   *
-   * Note: you must pass this function through js, setting a data attribute will not work!
-   *
-   * @example
-   * ```ts
-   * onLoaded: () => {
-   *   console.log('references loaded')
-   * }
-   * ```
-   */
-  onLoaded?: ((...args: unknown[]) => void) | undefined
-  baseServerURL?: string | undefined
-  /**
-   * List of servers to override the servers in the given OpenAPI document
-   *
-   * @default undefined
-   * @example [{ url: 'https://api.scalar.com', description: 'Production server' }]
-   */
-  servers?: OpenAPIV3_1.ServerObject[] | undefined
-  /**
-   * We’re using Inter and JetBrains Mono as the default fonts. If you want to use your own fonts, set this to false.
-   *
-   * @default true
-   */
-  withDefaultFonts?: boolean | undefined | unknown
-  /**
-   * By default we only open the relevant tag based on the url, however if you want all the tags open by default then set this configuration option :)
-   *
-   * @default false
-   */
-  defaultOpenAllTags?: boolean | undefined | unknown
-  /**
-   * Sort tags alphabetically or with a custom sort function
-   */
-  tagsSorter?: 'alpha' | ((a: Tag, b: Tag) => number) | undefined
-  /**
-   * Sort operations alphabetically, by method or with a custom sort function
-   */
-  operationsSorter?: 'alpha' | 'method' | ((a: TransformedOperation, b: TransformedOperation) => number) | undefined
-  /**
-   * Specifies the integration being used. This is primarily for internal purposes and should not be manually set.
-   *
-   * It’s used to:
-   * 1. Display debug information in the console.
-   * 2. Show a custom logo when importing OpenAPI documents in the Scalar App.
-   *
-   * Each supported integration has a unique identifier (e.g., 'express', 'nextjs', 'vue').
-   *
-   * To explicitly disable this feature, you can pass `null`.
-   *
-   * @private
-   */
-  _integration?:
-    | null
-    | 'adonisjs'
-    | 'docusaurus' // ✅
-    | 'dotnet'
-    | 'elysiajs'
-    | 'express' // ✅
-    | 'fastapi' // ✅
-    | 'fastify' // ✅
-    | 'go'
-    | 'hono' // ✅
-    | 'html' // ✅
-    | 'laravel'
-    | 'litestar'
-    | 'nestjs' // ✅
-    | 'nextjs' // ✅
-    | 'nitro'
-    | 'nuxt' // ✅
-    | 'platformatic'
-    | 'react' // ✅
-    | 'rust'
-    | 'vue' // ✅
-    | undefined
-  /**
-   * Whether to show the client button from the reference sidebar and modal
-   *
-   * @default false
-   */
-  hideClientButton?: boolean | undefined | unknown
-}
+export type ReferenceConfiguration = Partial<ApiReferenceConfiguration>
```

Comment:
@hanspagel we have type parity between the two now

Reply:
Do we need the old type though? If we modify the new type, we still want to support the old syntax, right?

Reply:
Sorry I dont quite understand? Its only there in case something is using it. The syntax is the same between both now?